src/tests/unit/tests/reports/package/footer-text-for-service.test.tsx

```diff
@@ -1,21 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { EnvironmentInfo } from 'common/environment-info-provider';
+import { EnvironmentInfoProvider } from 'common/environment-info-provider';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 import { FooterTextForService } from 'reports/package/footer-text-for-service';
 
 describe('FooterTextForService', () => {
     it('renders', () => {
-        const environmentInfo: EnvironmentInfo = {
-            extensionVersion: '1.2.3',
-            browserSpec: 'dummy browser version 1.0.1',
-            axeCoreVersion: '4.5.6',
-        };
+        const environmentInfoProvider: EnvironmentInfoProvider = new EnvironmentInfoProvider(
```

Comment:
Same here as in the other test: this should be a mock, not an instance of the actual class 'FooterText' has a dependency on.